src/cBiff12Writer.cls

```diff
@@ -127,7 +133,22 @@ Public Function Init( _
     ReDim m_uRowHdr.m_rgBrtColspan(0 To 0) As UcsBiff12BrtColSpanType
     m_uRowHdr.m_rgBrtColspan(0).m_colLast = m_lColCount - 1
     '--- success
-    Init = True
+    init = True
+End Function
+
+Public Function AddSheet(Optional ByVal SheetName As String = "", Optional ByVal lColCount As Long = 1) As Boolean
```

Comment:
`As String = ""` is redundant. Can you remove `= ""` please.

Reply:
Optional parameters will better with a default value, but sorry i cant change it now. Its  weekend and the first days at work we  have to do  end  of month jobs